IBPSA/Fluid/Movers/Preconfigured/UsersGuide.mo

```diff
@@ -0,0 +1,48 @@
+within IBPSA.Fluid.Movers.Preconfigured;
+package UsersGuide "User's Guide"
+  extends Modelica.Icons.Information;
+
+  annotation (preferredView="info",
+  Documentation(info="<html>
+<p>
+This package contains preconfigured versions of the mover models.
+They automatically configure a mover model based on
+<code>m_flow_nominal</code> and <code>dp_nominal</code>
+(and <code>speed_rpm_nominal</code> for <code>SpeedControlled_Nrpm</code> only).
+</p>
+<p>
+The configuration is as follows:
+</p>
+<ul>
+<li>
+The parameters <code>m_flow_nominal</code> and <code>dp_nominal</code>
+are used to construct the pressure curve <i>&Delta; p = f(m&#775;,  y)</i> where <i>m&#775;</i> is the mass flow rate and <i>y</i> is the speed.
+The curve support points come from regression of the pump and fan data records
+available in the data package.
+The pump pressure curve is based on all data records in
+<a href=\"Modelica://IBPSA.Fluid.Movers.Data.Pumps\">
+IBPSA.Fluid.Movers.Data.Pumps</a>
+and the fan pressure curve is based on all data records in
+<a href=\"Modelica://IBPSA.Fluid.Movers.Data.Fans\">
+IBPSA.Fluid.Movers.Data.Fans</a>.
+(See also
```

Comment:
Can you please be more specific what you mean by "all data records". Is it some average, or a typical curve?

Reply:
Hopefully this is clarified in improved documentation in ea34e88.

Reply:
Great, now it is clear.